Makefile

```diff
@@ -37,37 +34,81 @@ clean:: Makefile.coq.lib Makefile.coq
 	find . -type f \( -name '*~' -o -name '*.aux' -o -name '.lia.cache' -o -name '.nia.cache' \) -delete
 	rm -f Makefile.coq.lib Makefile.coq.lib.conf Makefile.coq Makefile.coq.conf _CoqProject
 
+EXTERNAL_DEPENDENCIES?=
+EXTERNAL_COQUTIL?=
+EXTERNAL_BEDROCK2?=
+
 COQUTIL_FOLDER := bedrock2/deps/coqutil
 BEDROCK2_FOLDER := bedrock2/bedrock2
 
+# Note: make does not interpret "\n", and this is intended
+DEPFLAGS_COQUTIL_NL=-Q $(COQUTIL_FOLDER)/src/coqutil coqutil\n
+DEPFLAGS_BEDROCK2_NL=-Q $(BEDROCK2_FOLDER)/src/bedrock2 bedrock2
```

Comment:
This line should probably end with `\n` too, as the variable name suggests?

Reply:
```suggestion
DEPFLAGS_BEDROCK2_NL=-Q $(BEDROCK2_FOLDER)/src/bedrock2 bedrock2\n
```